docs/design/multi-volcano-schedulers.md

```diff
@@ -1,4 +1,4 @@
-# Multi schedulers
+# Multi volcano schedulers
```

Comment:
Is multiple Volcano deployed in a cluster for scheduling?

Reply:
Yeah, and each volcano instance only cares about nodes that `node-selector` specified.